sdks/python/apache_beam/runners/worker/sdk_worker_main.py

```diff
@@ -190,7 +190,7 @@ def _get_worker_count(pipeline_options):
   future releases.
 
   Returns:
-    an int containing the worker_threads to use. Default is 12
+    an int containing the worker_threads to use. Default is 10000.
```

Comment:
Also update L206? (`return 12`)

Reply:
That was a mistake, in my follow-up #10091 I remove this method so I changed the comment to 12